### DISCUSSION TITLE
Ask user to confirm closing page with unsaved settings

### DISCUSSION THREAD

**Discussion by renpenguin** [stoatchat/discussions#545] [Feature Suggestions]
Currently if you're editing, say, a channel, when you click exit the editing view just closes, regardless of whether or not you saved. This can be frustrating if you forget to save a lot of changes. Some sort of "There are unsaved changes. Would you like to save?" dialogue would go a long way, and even making the Save button more defined, as currently it often blends in with the background and isn't noticed at all

**Comment by Bingobosal** [comment]
As someone who edited two times a role when creating a server and didn't notice the save button, I support this
